reference/orders.v3.yml

```diff
@@ -799,13 +799,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/MetaFieldCollectionResponse'
+              examples:
+                example-1:
+                  value:
+                    data:
+                      - permission_set: app_only
+                        namespace: 'minLength: 1'
+                        key: 'minLength: 1'
+                        value: 'minLength: 1'
+                        description: 'minLength: 0'
+                        resource_type: order
+                        resource_id: 'minimum: 0'
+                        id: 0
+                        created_at: '2019-08-24T14:15:22Z'
+                        updated_at: '2019-08-24T14:15:22Z'
+                    meta:
+                      pagination:
+                        total: 36
+                        count: 36
+                        per_page: 50
+                        current_page: 1
+                        total_pages: 1
+                        links:
+                          previous: string
+                          current: '?page=1&limit=50'
+                          next: string
```

Comment:
```suggestion
                      - permission_set: 'app_only'
                        namespace: 'Sale Department'
                        key: 'Staff Name'
                        value: 'Ronaldo'
                        description: 'Name of the sale staff'
                        resource_type: 'order'
                        resource_id: 101
                        id: 1
                        created_at: '2019-08-24T14:15:22Z'
                        updated_at: '2019-08-24T14:15:22Z'
                    meta:
                      pagination:
                        total: 1
                        count: 1
                        per_page: 50
                        current_page: 1
                        total_pages: 1
                        links:
                          previous: ''
                          current: '?page=1&limit=50'
                          next: ''
```

Reply:
Examples for order metafield endpoints should be in similar format with the product metafield ones: https://developer.bigcommerce.com/api-reference/bb4459a9412f6-get-all-product-metafields